src/dispatch/enums.py

```diff
@@ -5,6 +5,9 @@ class Visibility(str, Enum):
     open = "Open"
     restricted = "Restricted"
 
+    def __str__(self) -> str:
```

Comment:
Should we add this function to a parent class that the others would extend from so we don't have to copy and paste it in all of them?

Reply:
Sure, will make the change.

Reply:
Will address in separate PR.